packages/axelar-wasm-std/src/msg_id/mod.rs

```diff
@@ -45,6 +51,7 @@ pub enum MessageIdFormat {
     Base58TxDigestAndEventIndex,
     Base58SolanaTxSignatureAndEventIndex,
     HexTxHash,
+    AleoTransaction,
```

Comment:
I would prefer a more generic name that doesn't include Aleo if possible. Something like `Bech32TransactionIdAndTransitionId`